src/main/java/pages/DiffElementsPage.java

```diff
@@ -0,0 +1,65 @@
+package pages;
+
+import java.util.List;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+
+public class DiffElementsPage {
+    //for ex.2 test 6
+    @FindBy(xpath = "//label[text()[contains(., ' Water')]]/*[@type='checkbox']")
+    public static WebElement checkboxWaterElement;
+    @FindBy(xpath = "//label[text()[contains(., ' Wind')]]/*[@type='checkbox']")
+    public static WebElement checkboxWindElement;
+    //for ex.2 test 7
+    @FindBy(xpath = "//label[text()[contains(., ' Selen')]]/*[@type='radio']")
+    public static WebElement selenRadioButtonElement;
+    //for ex.2 test 8
+    @FindBy(className = "colors")
+    public static WebElement dropdownElement;
+    @FindBy(xpath = "//option[text()='Yellow']")
+    public static WebElement yellowElement;
+    @FindBy(xpath = "//*[@id=\"mCSB_2_container\"]/section/div/div/ul")
+    public WebElement logPanelElement;
+    //public List<WebElement> logPanelElement;
+    public static List<String> expectedTextInLogPanel = List.of(
+            //1
+            "Wind: condition changed to true",
```

Comment:
It's better to use some template here, like "%s: condition changed..." and parametrize what you need to verify in the test, using data provider for example